### PR TITLE
Catch or hatch patch

### PR DESCRIPTION
**Pull request by Leafsw0rd**
Adjusts all quest-text where Hatching is a valid strategy to mention it specifically. This patch at this point does not adjust any existing Capture or Hatch quests function, it just includes the hatch notifier if it's even slightly possible. Even cases where hatching is only theoretically possible with a lucky roll of the Shiny Code.

This also adjusts two pieces of text in the Manaphy Quest Line - it changes "Canalave Town" to it's correct name of "Canalave City" and changes a use of "Left" to "fled from" for clarity.

### CODE REVIEW

**Comment by Qwertypop04** [comment]
It isn't possible under any circumstance to hatch Zacian or Zamazenta. Those being hatchable is unintentional in the first place, but, because nobody had them before the quest was made, they can't be hatched for the quest

File: src/scripts/quests/questTypes/CaptureSpecificPokemonQuest.ts [hunk 1]
```diff
@@ -22,9 +22,9 @@ class CaptureSpecificPokemonQuest extends Quest implements QuestInterface {
             return this.customDescription;
         }
         if (this.amount === 1) {
-            return `Capture ${this.pokemon.name}.`;
+            return `Capture or hatch ${this.pokemon.name}.`;
         }
-        return `Capture ${this.pokemon.name} ${this.amount} times.`;
+        return `Capture or hatch ${this.pokemon.name} ${this.amount} times.`;
```
Comment (Aegyo) [inline src/scripts/quests/questTypes/CaptureSpecificPokemonQuest.ts:27]: `or hatch` is dependent on the includeBreeding parameter, you'll need to make the class remember it (in a new property), and reference here with
```ts
        return `Capture ${this.includeBreeding ? 'or hatch ' : ''}${this.pokemon.name} ${this.amount} times.`;
```

**Review by Aegyo — DISMISSED** [review]
It shouldn't be a problem but you have something weird going on with the commits listed here, did you make a branch off of one your own instead of the main repo develop one?

File: src/scripts/quests/questTypes/CaptureSpecificPokemonQuest.ts (hunk 1 above)
Comment (Leafsw0rd) [reply src/scripts/quests/questTypes/CaptureSpecificPokemonQuest.ts]: Incorporated in now.

**Comment by CypherX** [comment]
The addition of the _or hatch_ text may lead to confusion for new players that couldn't possibly have caught the pokemon yet. It may be worth making it dynamic and only include that text if the player has already caught the pokemon in question.

**Comment by Jaaslet** [comment]
> The addition of the _or hatch_ text (for the legendary quests) may lead to confusion for new players that couldn't possibly have caught the pokemon yet. It may be worth making it dynamic and only include that text if the player has already caught the pokemon in question.

I agree. And i don't think we need to make it dynamic, as most old players should had completed it by now. But yeah, that should be an other PR

**Review by Jaaslet — APPROVED** [review]
Looks good. Thanks for the PR

**Review by Jaaslet — APPROVED** [review]
Missedclicked... This should be an approval

**Pull request merged**